apel/db/loader/loader.py

```diff
@@ -17,10 +17,9 @@
 
 Module containing the Loader class.
 '''
+
 from __future__ import absolute_import
-from builtins import next
-from builtins import str
-from builtins import object
+from builtins import (next, str, object)
```

Comment:
Thanks for collating these, but the parentheses are unnecessary.

Reply:
I followed the example given in the https://python-future.org/overview.html# @tofu-rocketry. Thus, followed the same convention for better readability.

Reply:
Change to `from future.builtins import next,`...